search_indices/backends/faceted_attribute_backend.py

```
@@ -49,10 +49,10 @@ def get_attributes(
                 )
 
             if attributes:
-                attrs += map(
-                    lambda x, model=model: f"{str(model._meta.verbose_name)}_{x}",
-                    attributes["allowed"],
-                )
+                attrs += [
+                    f"{str(model._meta.verbose_name)}_{attribute}"
```

Ruff (0.16.1)

[warning] Use explicit conversion flag

Replace with conversion flag

(RUF010)